includes/template-functions.php

```diff
@@ -1011,8 +1006,7 @@ function sensei_the_single_lesson_meta() {
 			<?php do_action( 'sensei_frontend_messages' ); ?>
 
 			<?php
-			if ( ! $is_preview
-				|| Sensei_Utils::user_started_course( $lesson_course_id, get_current_user_id() ) ) {
+			if ( ! $is_preview || Sensei_Course::is_user_enrolled( $lesson_course_id ) ) {
```

Comment:
Just noting here that this entire block will probably be removed in this version. See https://github.com/Automattic/sensei/issues/2494#issuecomment-593866478